coriolis/tests/test_keystone.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import logging
+from unittest import mock
+
+from keystoneauth1 import exceptions as ks_exceptions
+
+from coriolis import exception
+from coriolis import keystone
+from coriolis.tests import test_base
+
+
+class KeystoneTestCase(test_base.CoriolisBaseTestCase):
+    """Collection of tests for the Coriolis Keystone module."""
+
+    def setUp(self):
+        super(KeystoneTestCase, self).setUp()
+        self.ctxt = mock.Mock()
+        self.ctxt.trust_id = None
+        self.ctxt.auth_token = 'test_token'
+        self.ctxt.project_name = 'test_project'
+        self.ctxt.project_domain_name = 'test_domain'
+        self.ctxt.user = 'test_user'
+        self.ctxt.project_id = 'test_project_id'
+        self.ctxt.roles = ['test_role']
+        self.connection_info = {
+            'auth_url': 'test_auth_url',
+            'username': 'test_username',
+            'password': 'test_password',
+            'project_name': 'test_project_name'}
+
+        self.trusts_auth_plugin = mock.Mock()
+        self.trusts_auth_plugin.auth_url = 'test_auth_url'
+        self.trusts_auth_plugin.get_user_id.return_value = 'test_trust_user_id'
+
+        self.auth = mock.Mock()
+        self.session = mock.Mock()
+        self.client = mock.Mock()
+        self.trust = mock.Mock()
+
+        self.trust.id = 'test_trust_id'
+        self.client.trusts.create.return_value = self.trust
+
+    @mock.patch('coriolis.keystone._get_trusts_auth_plugin')
```

Comment:
Global replace gone wrong :D 
This should have been the only valid `load_auth_from_conf_options` mock

Reply:
Yes, sorry for that. Thanks!